rx-preferences/src/main/java/com/f2prateek/rx/preferences2/RxSharedPreferences.java

```diff
@@ -19,6 +20,7 @@
 import static com.f2prateek.rx.preferences2.Preconditions.checkNotNull;
 
 /** A factory for reactive {@link Preference} objects. */
+@SuppressWarnings("WeakerAccess")
```

Comment:
Removes warnings about public APIs not being used, I wish Java had something similar to Kotlin's `@PublishedApi`...